### ISSUE TITLE
Version 0.7.16.70 fails to import

### ISSUE THREAD

**Issue by dlwyatt** [iainbrighton/PScribo#49]
Many errors, starting with this from Import-LocalizedData

```
Cannot find the Windows PowerShell data file 'PScribo.Resources.psd1' in directory
'C:\Users\dave.wyatt\Documents\WindowsPowerShell\Modules\pscribo\0.7.16.70\en-US\en-US\', or in any parent culture
directories.
```

(note the duplicated `en-US\en-US` bit)

**Comment by iainbrighton** [comment]
@dlwyatt DOH! Darn en-US systems 😉. Presumably [this is the culprit](https://github.com/iainbrighton/PScribo/blob/dev/PScribo.psm1#L9). I'll change this to `BaseDirectory = $PSScriptRoot;` and push an update (unless you think this won't solve the problem?

**Comment by dlwyatt** [comment]
Should work :)

**Comment by iainbrighton** [comment]
@dlwyatt Version 0.7.17 published to the Gallery which should resolve this. Let me know if it doesn't.

**Comment by it-praktyk** [comment]
@iainbrighton , I don't like how you implemented it.  I propose reopen and merge #50 for the next release.

**Comment by iainbrighton** [comment]
Closed with #64
